cmd/kube-controller-manager/controller-manager.go

```diff
@@ -105,8 +115,10 @@ func main() {
 			api.ResourceMemory: *nodeMemory,
 		},
 	}
-	nodeController := nodeControllerPkg.NewNodeController(cloud, *minionRegexp, machineList, nodeResources, kubeClient)
-	nodeController.Run(10 * time.Second)
+	if *syncNodes {
```

Comment:
Move if to above cloud var - cloud and nodeResources are only used for node controller.